web/src/edit/EditIntersectionPopup.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import { Intersection } from "../common/Intersection";
+
+  export let intersection: Intersection;
+</script>
+
+<div style="display: flex; align-items: center;">
+  {#if intersection.filter}
+    {#if intersection.hasAlreadyRotatedFilter}
+      <div class="icon">❌</div>
+      <img
+        src={`${import.meta.env.BASE_URL}/filters/diagonal_filter_icon.png`}
+        alt="Remove diagonal filter"
+        style="transform: rotate(45deg);"
+      />
+      Delete diagonal filter
+    {:else}
+      <div style="font-size: 200%; padding-bottom: 4px;" class="icon">↻</div>
+      <img
+        src={`${import.meta.env.BASE_URL}/filters/diagonal_filter_icon.png`}
+        alt="Rotate diagonal filter"
+        style="transform: rotate(45deg);"
```

Comment:
Rotating the icon rocks, I love this. I was trying to find examples of 4-ways at odd angles and wondering if we could dynamically rotate the icon to match what'll happen on the map exactly, but that's just an idea for later.

Reply:
We do have that exact angle - it's used when rendering on the map. I'll take a look at incorporating it here - TBH I don't think it matters much for this popup.

Reply:
Agreed, the popups are perfectly usable as they are -- thanks